roscube-i/nvidia_driver.rst

```diff
@@ -0,0 +1,102 @@
+.. _nvidia_driver:
+
+How to Install NVDIA Driver
+###########################
+
+The tutorial will guide you how to install nvidia driver.
+
+Supported Model: ROScube-I series
```

Comment:
ROScube-I series => ROScube-I-E
Only the extension box version can go with GPU.

Reply:
Done